cmd/sentry/app/app.go

```diff
@@ -197,3 +254,17 @@ func Run() {
 	}
 	log.Info("Sentry shut down gracefully")
 }
+
+// createOIDCTLSConfig creates a TLS configuration for the OIDC HTTP server
+func createOIDCTLSConfig(certFile, keyFile string) (*tls.Config, error) {
+	return &tls.Config{
+		GetCertificate: func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
+			cert, err := tls.LoadX509KeyPair(certFile, keyFile)
```

Comment:
Reading the TLS certificate on every inbound connection is inefficient- 

We should have a dynamic loader which waits for the file to be available, and then return that. We should also have an fs notify to watch for changes.

See https://github.com/dapr/kit/blob/main/crypto/spiffe/trustanchors/file/file.go

c